chapters/optimization/index.md

```diff
@@ -0,0 +1,155 @@
+## Flow Optmization
+
+##### Flow Optimization eases the process of rerunning Flows with different parameter sets. It allows out of the box hyperparameter tuning inside Jina.
+
+### Flow Optimization is hyperparameter tuning
+
+A common pattern when building a Flow in Jina is:
+
+1. Design the high level `flow.yml`
+2. Define several `pods.yml` files for all needed Executors
+3. Repeat until the evaluation metric suits your use case:
+  a. Change variable values in different Executors (e.g. used model, used model layer, details of the segmenter)
+  b. Index some data
+  c. Query some data and look at the results
+
+Flow Optimization automates step 3.
+It can be done via python code or JAML definitions as commonly used around Jina.
+
+### Before you start
+
+Make sure you install Jina via `Installation <https://docs.jina.ai/chapters/install/os/index.html>`_ with `jina[optimizer]`.
+
+Furthermore it is recommended to read *TODO: ADD LINK TO EVALUATOR SECTION*
+
+### Using FlowOptimization
+
+In this toy example, we try to find the optimal layer of an encoder for the final embedding.
+This is a common practice in machine learning.
+The best semantic representation for a given problem might not be the last layer of a given model.
+
+A Flow Optimization requires the following components:
+
+- At least one Flow and the corresponding Pod definitions via JAML
```

Comment:
Should we link important terms (Flow, Pod, etc) the first time they are mentioned to the Glossary?

Reply:
I won't do it yet. If you think about Flow Optimization, they should be familiar enough already. Anyhow, if we decide to do it in general, I'll add it for sure.